### DISCUSSION TITLE
Welcome to OpenGlobus Discussions!

### DISCUSSION THREAD

**Discussion by pavletto** [openglobus/openglobus#586] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by samvanemelen** [comment]
Hi everyone!

Big fan of the discussions board. I think it is a great place for questions and getting to know others in the community 😃.
My name is Sam, A freestyle developer from Belgium. I have plenty of passion projects in my spare time and I am a big nerd for game development and open source projects 🎮. I might also be posting as [@program-sam](https://github.com/program-sam) sometimes, which is where I share these passion projects.

I have been dreaming about a global-scale game for a while, and the open-sourceness and 3d rendering capabilities of openglobus makes me excited to join this community 🌐. I will slowly be exploring my ideas while contributing to features or issues I discover while playing around with openglobus.

Excited to see openglobus grow 😄

**Reply by ywszxdd** [reply]
I also want to jon it.

**Reply by Zemledelec** [reply]
Weclome!
